tests/func/test_live.py

```diff
@@ -237,6 +237,7 @@ def test_live_checkpoints_resume(
 
 def test_dvc_generates_html_during_run(tmp_dir, dvc, mocker, live_stage):
     show_spy = mocker.spy(dvc.live, "show")
+    webbrowser_open = mocker.patch("dvc.repo.live.webbrowser_open")
```

Comment:
@daavoo, it seems that the above change affected all of the tests and now opens the browser while running the tests.